cpanfile

```diff
@@ -44,6 +44,7 @@ requires 'base';
 requires 'constant';
 requires 'strict';
 requires 'warnings';
+requires 'Socket::MsgHdr';
```

Comment:
at least we have a package available for it

Reply:
FYI the package generates its class methods inside the INIT block which doesn't work with something we are doing:
```
Too late to run INIT block at /usr/lib/perl5/vendor_perl/5.26.1/x86_64-linux-thread-multi/Socket/MsgHdr.pm line 15.
Too late to run INIT block at /usr/lib/perl5/vendor_perl/5.26.1/x86_64-linux-thread-multi/Socket/MsgHdr.pm line 59.
```